oauth2-server/src/main/java/com/clouway/oauth2/IssueNewTokenActivity.java

```diff
@@ -57,14 +57,21 @@ public Response execute(Client client, Identity identity, Set<String> scopes, Re
       Pem.Block key = certificates.get(certKey);
       String host = request.header("Host");
 
+      Map<String, Object> claims = new LinkedHashMap<>();
+      claims.put("iss", host);
+      claims.put("aud", client.id);
+      claims.put("sub", identity.id());
+      claims.put("iat", instant.asDate());
+      claims.put("exp", new Date(accessToken.expirationTimestamp()));
```

Comment:
I suppose not, as it has to be in milliseconds. 

The BearerToken class has method about this: 

```java
 /**
   * Gets time to live in seconds of the current token.
   *
   * @param instant the instant time used for check
   * @return the time to live in seconds
   */
  public Long ttlSeconds(DateTime instant) {
    return (expiresAt.timestamp() - instant.timestamp()) / 1000;
  }
```

which calculates the time to live (exp) in seconds from the current time.